### PR TITLE
Issue #1987066 by Kevin Morse, JohnAlbin, rteijeiro, echoz, ry5n: Rename files to match CSS file naming convention.

### PR DESCRIPTION
**Pull request by hellomrcat**
PR for https://github.com/backdrop/backdrop-issues/issues/207

### CODE REVIEW

File: core/modules/block/block.module [hunk 1]
```diff
@@ -139,7 +139,7 @@ function block_menu() {
       'access arguments' => array($key),
       'theme callback' => '_block_custom_theme',
       'theme arguments' => array($key),
-      'file' => 'block.admin.inc',
+      'file' => 'css/block.admin.inc',
```
Comment (quicksketch) [inline core/modules/block/block.module]: This entire set of changes in `block_menu()` shouldn't be added. I think these path changes were just accidentally changed, as these .inc files shouldn't be affected by CSS changes.

File: core/modules/field/field.info [hunk 2]
```diff
@@ -6,4 +6,4 @@ backdrop = 1.x
 dependencies[] = field_sql_storage
 dependencies[] = entity
 required = TRUE
-stylesheets[all][] = theme/field.css
+stylesheets[all][] = css/field.css
```
Comment (quicksketch) [inline core/modules/field/field.info:9]: Looks like this path should be `css/field.module.css`.

**Comment by quicksketch** [comment]
A few files got renamed here that shouldn't have:

```
 core/modules/book/book.admin.inc → core/modules/book/css/book.admin.inc 
```

.inc files contain PHP code and I think this patch intended to move only CSS files. Let's roll back the moving of the .inc files and just move the .css files.

**Comment by hellomrcat** [comment]
@quicksketch fixed!

**Comment by quicksketch** [comment]
I'm going to assign this to @jenlampton to review, as she's the master of theme-system changes.

**Comment by jenlampton** [comment]
@hellomrcat there are a few name changes in here that are different between Backdrop and Drupal 8.

In Drupal 8, files like taxonomy.css got renamed to taxonomy.module.css, but we're not going to add the .module in Backdrop, as per our [CSS coding standards](http://backdropcms.org/develop/css-standards#file-names)

Can you please do a second pass on this issue and un-do the renames that include .module.

**Comment by hellomrcat** [comment]
@jenlampton fixed!

**Comment by quicksketch** [comment]
Wow, great job! I'll review this one manually since we don't have any testing on the appearances of things.

**Comment by quicksketch** [comment]
This is fantastic. I moved admin_menu.css and comment.css into css directories as well, since they were the only remaining non-directoried CSS files left in core modules.

**Pull request merged**